blueocean-rest-impl/src/main/java/io/jenkins/blueocean/service/embedded/rest/QueuedFreeStyleRun.java

```diff
@@ -0,0 +1,13 @@
+package io.jenkins.blueocean.service.embedded.rest;
+
+import io.jenkins.blueocean.rest.annotation.Capability;
+import io.jenkins.blueocean.rest.hal.Link;
+
+import static io.jenkins.blueocean.rest.model.KnownCapabilities.JENKINS_FREE_STYLE_BUILD;
+
+@Capability(JENKINS_FREE_STYLE_BUILD)
+public class QueuedFreeStyleRun extends QueuedBlueRun {
```

Comment:
I guess `QueuedFreeStyleRun` is added so that client can distinguish free style vs pipeline queue using capability? Don't see client using it though.